packages/paste-website/src/pages/components/alert-dialog/index.mdx

```diff
@@ -0,0 +1,209 @@
+
+import {graphql} from 'gatsby';
+// import Changelog from '@twilio-paste/alert-dialog/CHANGELOG.md';
+import {Anchor} from '@twilio-paste/anchor';
+import {AlertDialog} from '@twilio-paste/alert-dialog';
+import {Button} from '@twilio-paste/button';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {defaultExample, destructiveExample} from '../../../component-examples/AlertDialogExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/alert-dialog"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/alert-dialog/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Alert Dialog"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/alert-dialog"
+  storybookUrl="/?path=/story/components-alert-dialog--alert-dialog-with-two-actions"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for use in production and components like Alert Dialog won't likely see any API changes.
+    The term "Alpha" means there is only basic documentation at this time. Please share your experience using this Alpha
+    component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new" showExternal>
+      starting a discussion on GitHub
+    </Anchor>
+    .
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Alert Dialog
+
+The Alert Dialog should be used when a user flow needs to be stopped unless a specific action is taken. It is used to convey
+important messages that can't be ignored like [Alert](/components/alert) content. Potential use-cases for Alert Dialog are:
+
+- Confirming a permanent change, like deleting user-made data
+- Relaying an important system message like a maintenance downtime window
+
+While visually similar to the [Modal](/components/modal), the Alert Dialog ensures the user acknowledges the information in the dialog.
+Users cannot interact with content outside an active Alert Dialog window until the user explicitly closes it.
+When the Alert Dialog opens, focus moves to an element contained in it (by default, the first element that can be focused),
+and the focus should stay in and cycle through the Alert Dialog's content. Focus shouldn't return
+to the underlying page until the user closes the Alert Dialog. This can happen in any of the following ways:
```

Comment:
```suggestion
Though it's visually similar to a [Modal](/components/modal), an Alert Dialog requires the user to acknowledge the information in the dialog.

Users cannot interact with content outside an active Alert Dialog window until the user explicitly closes it. When the Alert Dialog opens, focus moves to an element contained in it (by default, the first element that can be focused), and the focus should stay in and cycle through the Alert Dialog's content. Focus shouldn't return to the underlying page until the user closes the Alert Dialog. This can happen in any of the following ways:
```